cryosparc/api.py

```diff
@@ -0,0 +1,409 @@
+import json
```

Comment:
FYI @Puratinamu @notLiria this was largely moved from the cryosparc repo with minimal modifications, I'll point out the main differences below.